### ISSUE TITLE
slackin broken

### ISSUE THREAD

**Issue by jmimico** [overture-stack/website#210]
Current implementation of slackin is broken. Needs a new approach or refactor to get going again.

**Comment by caravinci** [comment]
Disabled the "Join us on slack" button on the website temporarily.

**Comment by caravinci** [comment]
TL;DR: Re-enabling the button, as a solution was found

We found a built image of the `docker-slackin` app we rely upon (which due to the app's dependencies being outdated, doesn't build locally). This is a good interim solution, and if we ever need to get it working, we can fork the repo and build our own.
In the meanwhile, the app runs like so `root@slackin/: docker run --env-file env.list --name slackin --restart=always -d -p 80:3000 viktorbenei/docker-slackin`

For future reference, the relevant repos:
- docker-slackin https://github.com/viktorbenei/docker-slackin
- slacking itself https://github.com/rauchg/slackin
